src/index.d.ts

```diff
@@ -17,6 +17,10 @@ export interface TrackingProp<P = {}> {
   getTrackingData(): {};
 }
 
+export interface UseTrackingProp<P = {}> extends TrackingProp<P> {
```

Comment:
@olesyd Didn't we already add this?

Reply:
As I see we don't have the type `UseTrackingProp`. But there is a strange thing that can't figure out why does it happen. When I build Ads project with the `react-tracking` of 9.0.2 version (previous version with `prop-types` in devDependencies) `UseTrackingProp` exists in the `node_modules/@spotim/react-tracking/build/index.d.ts` but with 9.0.3 version `UseTrackingProp` is missing out and I get error `Property 'Track' does not exist on type 'TrackingProp<{}>'`